build/AzurePipelinesTemplates/WindowsAppSDK-RunTestsInPipeline-Job.yml

```diff
@@ -0,0 +1,120 @@
+# This runs the integration tests directly in the pipeline on the images hosted by the 'WinAppSDK-Test-Pool'.
+# This pool is a 1ES Hosted Pool with custom images, which lets us run tests on builds beyond what is
+# supported by Helix or the Azure DevOps Microsoft hosted agents. In case of arm64, we use the 
+# 'WinAppSDK-Test-Pool-Arm64' pool instead, and use parameter isArm64Platfrom to drive pool selection.
+
+parameters:
+- name: jobName
+  type: string
+  default: PipelineTests
+- name: isArm64Platfrom
+  type: boolean
+  default: false
+- name: dependsOn
+  type: object
+  default:
+  - ''
+  
+jobs:
+- job: ${{ parameters.jobName }}
+  dependsOn: ${{ parameters.dependsOn }}
+  strategy:
+    # Debug/Release buildConfiguration below refers to the build of the tests/apps. The OS image bits are always Release build.
+    matrix:
+      # The test matrix for x64 is currently larger; this is where we exercise most variations.
+      ${{ if not( parameters.isArm64Platfrom ) }}:
+        RS5_x86fre:
+          imageName: Windows.10.Enterprise.RS5
+          buildPlatform: x86
+          buildConfiguration: release
```

Comment:
Is there any reason why only "release" config is being tested?
Would it be feasible at this point to exercise the "debug" config for a few OS versions? And similar for the smaller arm64 group?

Reply:
We are currently only building for release on the Build job. [https://github.com/microsoft/WindowsAppSDK/blob/73b7848a2acac05d310d6c5730b744a356f44480/build/ProjectReunion-BuildFoundation.yml#L53](url)

Reply:
I'm ok with not introducing the debug config in this PR.